### PR TITLE
open-cluster-management/backlog#12397 -- app sets info for dev review

### PR DESCRIPTION
**Pull request by swopebe**
--more about app sets with argo

### CODE REVIEW

File: applications/app_console.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ See the following application console capabilities:
 * Visualize deployed applications across your clusters, including any associated resource repositories, subscriptions, and placement configurations.
 
 * Create and edit applications, and subscribe resources. By default, the hub cluster can manage itself and is named the `local cluster`. You can choose to deploy application resources to this local cluster, though deploying applications on the local cluster is not best practice. 
-//is this still an issue?
+//is this still an issue?--please confirm this line
```
Comment (mikeshng) [inline applications/app_console.adoc]: Yes, its still an issue. Keep this.

**Review by mikeshng — COMMENTED** [review]
I don't have enough expertise to comment on the ApplicationSet related topics.

**Review by fxiang1 — APPROVED** [review]
Looks good!

**Comment by swopebe** [comment]
Thanks so much @mikeshng and @fxiang1 ! Merging and closing.

**Pull request merged**